sip-quickstart/assets/admin/index.html

```diff
@@ -5,7 +5,14 @@
   </head>
   <body>
     <div id="app">
-      <div class="login" v-if="!adminClient.isReady">
+      <div v-if="!passwordChangedSuccessfully && errorMessage">
+        <h1>Change your admin password</h1>
+          <div v-html="errorMessage"></div>
+          <p>
+            The password must be a minimum of 12 characters, contain at least 1 digit, and have mixed case.
+          </p>
+      </div>
+      <div class="login" v-if="!adminClient.isReady && passwordChangedSuccessfully">
```

Comment:
Wondering if setting to `isReady` will still require that passwordChangedSuccessfully

Reply:
I think probably not. Removed! Thanks!